File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    var odd=0;
    var even=0;
    var answer = 0;
    num_list.map((item,index)=>{
        if(index%2===0){
          odd += item 
        }else{
          even += item
        }
    }
)
    return Math.max(odd, even)
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int oddResult = 0;
        int evenResult = 0;
        for(int i =0; i< num_list.length; i++){
            if(i % 2 ==0){
                evenResult += num_list[i];
            }else{
                oddResult += num_list[i];
            }
        }
        return oddResult > evenResult ? oddResult : evenResult;
    }
}